recipe/meta.yaml

```diff
@@ -7,15 +7,17 @@ package:
 source:
   fn: jsoncpp-{{ version }}.tar.gz
   url: https://github.com/open-source-parsers/jsoncpp/archive/{{ version }}.tar.gz
-  sha256: 7c285fc40ad0c113e436a1271c4e38b5017b5c7782c306e90be9d6b2ffa90212
+  sha256: 858db2faf348f89fdf1062bd3e79256772e897e7f17df73e0624edf004f2f9ac
+  patches:
+    # https://github.com/open-source-parsers/jsoncpp/pull/632
+    - vc9-stdint.patch
```

Comment:
So to resolve this we could actually just include `msinttypes` as a `build` dependency.